helix-query.yaml

```diff
@@ -0,0 +1,12 @@
+version: 1
+
+indices:
+  articles:
+    include:
+      - '/articles/**'
```

Comment:
@Lauraa1003  Do we want to keep articles ? or name it blog ?

Reply:
Named it blog